src/main/java/org/carlmontrobotics/subsystems/IntakeShooter.java

```diff
@@ -202,8 +202,12 @@ public void setMaxIntake(int direction) {
 
     }
 
-    public void setMaxOuttake(int direction) {
-        outtakeMotorVortex.set(1 * direction);
+    public void setMaxOuttake(int d) {
+        outtakeMotorVortex.set(1 * d);
+    }
+
+    public void setAutoRuinerOuttake(double d) {
```

Comment:
The method name suggests that this can only be used for the auto-ruiners, but you are just setting the speed of the outtake motor which could have other applications. You may want to rename the method name.

Reply:
I don't think we should change it unless there are other things we can apply it to

Reply:
Actually, on line 94-96 there is a method called `motorSetOuttake(double speed)` which does the same exact thing as  your `setAutoRuinerOuttake(double d)`. You should just use the former method.

Reply:
ok